examples/3d-tiles/babel.config.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module.exports = function babelConfig(api) {
+  api.cache(() => process.env.NODE_ENV === 'production'); // eslint-disable-line
+
+  return {
+    presets: ['@babel/preset-env'],
+    plugins: [['@babel/plugin-proposal-class-properties', {loose: false}]]
```

Comment:
I can't see where you use this. Generally I prefer staying with standard ES2018. If you really need it then OK, but if just to save one `.bind(this)` then I am not sure it is worth it.